certbot/updater.py

```diff
@@ -29,8 +29,10 @@ def run_generic_updaters(config, lineage, plugins):
         return
     try:
         # installers are used in auth mode to determine domain names
-        installer, _ = plug_sel.choose_configurator_plugins(config, plugins, "certonly")
-    except errors.PluginSelectionError as e:
+        installer = plug_sel.get_installer(config, plugins)
+    except (errors.MissingCommandlineFlag,
+            errors.MisconfigurationError,
+            errors.PluginSelectionError) as e:
```

Comment:
Unless you're sure you listed all the error types here and there are errors we may throw which you don't want to catch, I'd just use `errors.Error` here.

Reply:
`errors.Error` is much more readable, using that instead as all of the errors are handled in a similar (non-interrupting) fashion here anyway.